R/user_id.R

```diff
@@ -0,0 +1,78 @@
+#' Mark a user id as a screen name
+#' 
+#' @description 
+#' There are two ways to identify a twitter user: a screen name (e.g.
+#' "justinbieber") or a user identifier (e.g. "27260086"). User identifiers
+#' are 64-bit integers which can't be represented natively by any base R data
+#' structures, so in most cases rtweet returns them as strings. This introduces
+#' an ambiguity, because user names can also consist solely of numbers 
+#' (e.g. "123456"). You can resolve this ambiguity by using `as_screenname()` 
+#' to a character vector a screen name.
```

Comment:
I think this section can be made more easier to understand. I would delete this sentence: "User identifiers are 64-bit integers which can't be represented natively by any base R data structures, so in most cases rtweet returns them as strings."
First, I hope we can return them consistently as strings (I think this is the case already), second to the user it doesn't matter what/how R works all important here, imho, is how to get rtweet to work. If you want to mention it I would make a note or place it at the end. 

Also it doesn't have a `#' @return` documentation even if trivial it will be helpful.

Reply:
I think it's worth noting that user ids look like numbers, but you can't store them in R as numbers. But I agree that comes too soon and most users won't care, so I'll move it later and make it a parenthetical.